app/src/main/res/values/strings.xml

```diff
@@ -621,12 +621,12 @@
     <string name="overwrite_unrelated_warning">A file with this name already exists</string>
     <string name="overwrite_finished_warning">A downloaded file with this name already exists</string>
     <string name="overwrite_failed">cannot overwrite the file</string>
-    <string name="download_already_running">There is a download in progress with this name</string>
-    <string name="download_already_pending">There is a pending download with this name</string>
+    <string name="download_already_running">Downloading something with this name</string>
+    <string name="download_already_pending">Waiting to download something with this name</string>
```

Comment:
```suggestion
    <string name="download_already_pending">Waiting to download something with this name…</string>
```

Reply:
Also here, this is different, what's wrong with the old string?

Reply:
"Pending" and "a download" is more complex.